.github/workflows/formatting.yaml

```diff
@@ -0,0 +1,32 @@
+name: Cloud Hypervisor Code Formatting
+on: [pull_request, merge_group]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Code Formatting
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - nightly
+        target:
+          - x86_64-unknown-linux-gnu
+          - aarch64-unknown-linux-musl
```

Comment:
Great work! BTW why are we having `gnu` for `x86_64` and `musl` for `riscv64` :)

Reply:
These are the most mature toolchains for those two and I didn't want to waste resources with a full set.